demo/kubernetes/rook-operator-with-repo.yaml

```diff
@@ -11,11 +11,12 @@ spec:
     spec:
       containers:
       - name: rook-operator
-        image: quay.io/myrepo/rook-operator:mytag
+        image: quay.io/myrepo/rookd:mytag
```

Comment:
I dont think we need a new yaml file example to illustrate this. Perhaps we can add the extra env to the original rook-operator.yaml commented out and comments just like you did for the rook-storageclass.yaml. I think it would confuse people.